src/codegen/generate.ts

```diff
@@ -283,6 +288,9 @@ export default class ApiGenerator {
     public readonly isConverted = false,
   ) {}
 
+  // Store parent schemas (see `preprocessSchemas` for the definition of a parent schema)
+  parentSchemaRefs: Set<string> = new Set();
```

Comment:
Can we call this `anyOfOneOfComponents`?

Reply:
I agree that "parent schema" is too general and I'm open to using a more suitable term if there is one. However, I feel `anyOfOneOfComponents` is kind of confusing. Shouldn't the variable represent components that have a `discriminator` but don't have `oneOf` or `anyOf`?

Reply:
Looking at other field variables, "refs" seems to be used to mean `ts.TypeReferenceNode`. Therefore, the name `parentSchemaRefs` could be a bit misleading. I'm going to change it to `parentSchemas`. As for "parent schema", I haven't come up with a better idea yet.